doc/md/guides/mysql/generated-columns.md

```diff
@@ -161,7 +161,7 @@ Select all the triangles in the table using `SELECT  *  FROM triangles` to recei
 | 3    | 4    | 5                  |
 | 6    | 8    | 10                 |
 
-### Implementing Virtual Generated Columns Columns with Atlas
+### Implementing Virtual Generated Columns with Atlas
```

Comment:
I'm assuming this was an unintentionally duplicated word, but I could be wrong....